astropy/time/core.py

```diff
@@ -1354,6 +1354,82 @@ def sort(self, axis=-1):
         return self[self._advanced_index(self.argsort(axis), axis,
                                          keepdims=True)]
 
+    def mean(self, axis=None, dtype=None, out=None, keepdims=False, *, where=True):
+        """Mean along a given axis.
+
+        This is similar to :meth:`~numpy.ndarray.mean`, but adapted to ensure that the full precision given by the two
+        doubles ``jd1`` and ``jd2`` is used, and that corresponding attributes are copied.
+
+        Note that the ``out`` argument is present only for compatibility with ``np.mean``; since `Time` instances are
+        immutable, it is not possible to have an actual ``out`` to store the result in.
+
+        Similarly, the ``dtype`` argument is also present for compatibility with :meth:`~numpy.ndarray.mean` and the
+        ``dtype`` is determined by the ``Time`` format. An :exc:`~ValueError` is raised if this is set to anything
+        other than `None`.
+
+        Parameters
+        ----------
+        axis : None or int or tuple of ints, optional
+            Axis or axes along which the means are computed. The default is to
+            compute the mean of the flattened array.
+        dtype : None
+            Only present for compatibility with :meth:`~numpy.ndarray.mean`, must be :class:`~None`.
+        out : None
+            Only present for compatibility with :meth:`~numpy.ndarray.mean`, must be :class:`~None`.
+        keepdims : bool, optional
+            If this is set to True, the axes which are reduced are left
+            in the result as dimensions with size one. With this option,
+            the result will broadcast correctly against the input array.
+
+            If the default value is passed, then ``keepdims`` will not be
+            passed through to the ``mean`` method of sub-classes of
+            `ndarray`, however any non-default value will be.  If the
+            sub-class' method does not implement ``keepdims`` any
+            exceptions will be raised.
+
+        where : array_like of bool, optional
+            Elements to include in the mean. See :func:`~numpy.ufunc.reduce` for details.
```

Comment:
It is weird that reduce API doc linking failed and I really don't know if removing `:func:` would fix anything but it is listed as "method" on https://numpy.org/doc/stable/reference/generated/numpy.ufunc.reduce.html , similar to https://numpy.org/doc/stable/reference/generated/numpy.ndarray.mean.html . I would suggest you try to build the docs locally and see if you can figure it out there. Maybe inspecting https://numpy.org/doc/stable/objects.inv would give you a clue but I am not sure also.

```suggestion
            Elements to include in the mean. See `~numpy.ufunc.reduce` for details.
```

Reply:
Maybe solve it by linking to `np.mean` instead?

Reply:
@pllim, that seemed to work! Good catch that they listed it as a method for some reason.

@mhvk, I considered linking to `np.mean`, but it would be more direct to link to `np.ufunc.reduce` since that's where `np.mean` leads you.